xxxx-release-and-versioning.md

```diff
@@ -0,0 +1,87 @@
+# OIP-xxxx: eWallet Release & Versioning
+
+## Abstract
+
+This document specifies a procedure for releasing and versioning to allow for stable production environment. OmiseGO eWallet Server (hereafter "eWallet Server") is meant to be deployed by a various entities, which may range from a personal store to an enterprise (hereafter "Provider") and thus require a stable operation and smooth upgrade path that maximizes compatibilities.
+
+## Overview
+
+The eWallet release is consist of a stable release and a development release. A stable release is meant to be run by a Provider that require stability and compatibility, while a development release is meant for development and testing propose. The stable releases should be versioned using [SemVer](https://semver.org/).
+
+## Specifications
+
+### Stable Release
+
+A stable release is a release that provides compatibility and stability guarantee and must pass an extensive end-to-end testing and migration testing from supported versions prior to release.
+
+| Test subject        | Required |
+| ------------------- | -------- |
+| Linting             | **Yes**  |
+| Unit testing        | **Yes**  |
+| Integration testing | **Yes**  |
+| End to end testing  | **Yes**  |
+| Migration testing   | **Yes**  |
+
+A stable release will be provided as a Docker image based on Alpine Linux. The Docker image will be tagged with the release version (e.g. `v1.0.3`) and will use a `:latest` tag to refer to the latest stable release.
```

Comment:
No binary release for stable?

Reply:
I thought about it, then decided that stable should be Docker-only for now (as well as maybe nightly). We simply don't have enough manpower to build and test more than single platform right now, and releasing an official binary for N platforms is a little bit complicated. We might do in the future though.